manual/Makefile

```diff
@@ -1,9 +1,17 @@
 # Makefile for the reference and user manuals
 TEXDIR=tex_files
 VPATH=$(TEXDIR)
-.PHONY:all
+.PHONY:all clean sphinx
 
-all: user_manual.pdf \
+all: sphinx
+
+sphinx:
+	sphinx-build -b latex sphinx/ build/
+	cd build && latexmk -pdf BOUT
+	test -e BOUT.pdf || ln -s build/BOUT.pdf .
+	@echo "Documentation is available in BOUT.pdf"
+
+old: user_manual.pdf \
```

Comment:
There is a `sphinx-docs-latex` target in [`make.config`][1] that could just point to this, maybe. Either that or renamed to something more memorable.

[1]: https://github.com/boutproject/BOUT-dev/blob/v4.1-RC/make.config.in#L361